test/BUILD.gn

```diff
@@ -199,6 +200,10 @@ test("brave_unit_tests") {
     "//services/preferences/public/cpp",
   ]
 
+  if (unstoppable_domains_enabled) {
```

Comment:
tests should go in brave/browser/unstoppable_domains and a dep should be added here